rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -1210,6 +1222,145 @@ fn test_icrc3_get_blocks_number_of_blocks_limit() {
     check_icrc3_get_block_limit(vec![(0, 1), (0, 100)]);
 }
 
+#[cfg(not(feature = "u256-tokens"))]
```

Comment:
Thanks for adding this test!

nit: Ideally this test should work with both token types, which would require building the wasm for both token types with and without the features. On the other hand, I don't see anything here that could change for `u64` vs `u256`, so I guess just testing with `u64` would be acceptable.

Reply:
The token type does not change the behaviour of the two endpoints, since there are no tokens returned. Only an index (u64) and a hash.